src/Sulu/Component/Content/Document/Subscriber/Compat/ContentMapperSubscriber.php

```diff
@@ -87,7 +87,7 @@ public static function getSubscribedEvents()
     {
         return [
             Events::REMOVE => [
-                ['handlePreRemove', 500],
+                ['handlePreRemove', 550],
```

Comment:
would not set it to the same as the other subscribers else we could run there into the same problem:

```
bin/console sulu:document:subscriber:debug remove
```

Would use in this case 510.

Reply:
Sure will do. In my local test it did not make any problems as those subscribes "do things" in their scope. And I wasn't sure if you want to stick to +-50 steps.